File: EulerProject/Euler026.py
from os import *
from shutil import *
from string import *
import time
from decimal import Decimal


starttime = time.time()

for x in range(1,50,2):
	print float(1.0/x)

print Decimal(1.0/9.0)

print "Execution time ", time.time() - starttime

File: EulerProject/Euler026.java
import java.util.*;

class Euler026
{
	public static void main(String[] args)
	{
		long startTime = System.currentTimeMillis() ;
		
		//double x = 0.0 ;
		
		for ( int x = 7 ; x < 100 ; x+=2 )
		{
			double dummy = x ;
			System.out.println("X :" + dummy +" \t Fraction:" +  1.0/dummy);
		}
	



		double execTime = (System.currentTimeMillis() - startTime) ;
		System.out.print("Total execution time is :");
		System.out.print(execTime/1000);
		System.out.println(" sec/s");
	//	Scanner in = new Scanner(System.in);
	//	in.nextLine();
		
	}
}


